package.json

```diff
@@ -14,7 +14,7 @@
     "pi": "yarn && NODE_ENV=pi node server.js",
     "local": "yarn && cross-env NODE_ENV=local node server.js",
     "prepush": "yarn lint && yarn test",
-    "startDemo": "yarn && cross-env NODE_ENV=demo node server.js"
+    "startDemo": "yarn && cross-env NODE_ENV=demos node server.js"
```

Comment:
demo c'est un reserved word, ca me donne tjr un erreur de ressource in use quand j'utilise demo comme node-env